File: 2271-rearrange-array-elements-by-sign/solution.java
class Solution {
    public int[] rearrangeArray(int[] nums) {
        ArrayList<Integer> list1=new ArrayList<>();
        ArrayList<Integer> list2=new ArrayList<>();
        for(int i=0;i<nums.length;i++){
            if(nums[i]>0){
                list1.add(nums[i]);
            }
            else{
                list2.add(nums[i]);
            }
        }
        int[] arr=new int[nums.length];
        for(int i=0;i<list1.size();i++){
            arr[i*2]=list1.get(i);
            arr[2*i+1]=list2.get(i);
        }
        return arr;
    }
}

File: 2271-rearrange-array-elements-by-sign/solution.py
class Solution:
    def rearrangeArray(self, nums: List[int]) -> List[int]:
        pos=[i for i in nums if i>0]
        neg=[i for i in nums if i<0]
        res=[]
        for i in range(0,len(pos)):
            res.append(pos[i])
            res.append(neg[i])

        return res
        
